basedtyping/runtime_checks.py

```diff
@@ -0,0 +1,13 @@
+from types import UnionType
+from typing import Sequence, cast
+
+
+def is_subclass(type1: type | UnionType, type2: type | UnionType) -> bool:
```

Comment:
I wonder if `issubtype` would be a better name?

Reply:
`issubtype`? also whhats ur opinion on `is_subtype` vs `issubtype`, nothing in this language is consistent so what do we do?

Reply:
`issubform` is a good one imo. kingsised form of `form` is `typeform` means as 'type time type' as opposed to `type` meaning as 'runtime class'

Reply:
renamed to `issubform`